src/Commands/AcmsBuildCommand.php

```diff
@@ -87,15 +86,21 @@ public function __construct(
   /**
    * {@inheritdoc}
    */
-  protected function configure() :void {
+  protected function configure(): void {
     $this->setName("acms:build")
       ->setDescription("Use this command to build composer dependencies.")
-      ->setDefinition([
-        new InputArgument('name', NULL, "Name of the starter kit"),
-        new InputOption('uri', 'l', InputOption::VALUE_OPTIONAL, "Multisite uri to setup drupal site.", 'default'),
-        new InputOption('generate', 'ge', InputOption::VALUE_NONE, "Create build.yml file without running composer install/require."),
-      ])
-      ->setHelp("The <info>acms:build</info> command to build composer dependencies & downloads it based on user selected use case.");
+      ->addArgument('name', InputArgument::OPTIONAL, "Name of the starter kit")
+      ->addOption('uri', 'l', InputOption::VALUE_OPTIONAL, "Multisite uri to setup drupal site.", 'default')
+      ->addOption('generate', 'ge', InputOption::VALUE_NONE, "Create build.yml file without running composer install/require.");
+
+    // Get acquia CMS build questions.
+    $buildQuestions = $this->acquiaCmsCli->getOptions('build');
+    // Iterate build questions to prepare for input option method.
+    foreach ($buildQuestions as $option => $value) {
+      $this->addOption($option, '', InputOption::VALUE_OPTIONAL, $value['description'], $value['default_value']);
+    }
+
+    $this->setHelp("The <info>acms:build</info> command to build composer dependencies & downloads it based on user selected use case.");
```

Comment:
@vishalkhode1 @apathak18 can we update the help text.
```suggestion
    $this->setHelp("The <info>acms:build</info> command facilitates the compilation of composer dependencies, downloading them according to the user's selected use case.");
```

Reply:
`command to build composer dependencies & downloads it based on user selected use case.`
This is more simpler